.toys/new-library.rb

```diff
@@ -118,6 +125,47 @@ def call_owlbot
   exec_tool cmd
 end
 
+def create_release_please_configs
+  manifest = JSON.parse File.read manifest_name
+  manifest[gem_name] = "0.0.1"
+  manifest = sort_hash add_fillers manifest
+  File.write manifest_name, "#{JSON.pretty_generate manifest}\n"
+
+  config = JSON.parse File.read config_name
+  config["packages"][gem_name] = {
+    "component" => gem_name,
+    "version_file" => gem_version_file
+  }
+  config["packages"] = sort_hash config["packages"]
```

Comment:
Can we use `sort_by` method over here? Like `config["packages"].sort_by {|k,v| k}.to_h`

Reply:
You're right, that's a lot better. Good call!

Reply:
Wasn't very sure about the key type, generally symbol type keys dont work in `sort` method.

Reply:
Ruby `Symbol`s are comparable. Wasn't _always_ the case, but has been since Ruby 1.9 if I recall.